kernel/kernel-keymanager-service/src/main/java/io/mosip/kernel/keymanager/hsm/impl/pkcs/PKCS12KeyStoreImpl.java

```diff
@@ -126,7 +133,9 @@ public PKCS12KeyStoreImpl(Map<String, String> params) throws Exception {
         this.asymmetricKeyAlgorithm = params.get(KeymanagerConstant.ASYM_KEY_ALGORITHM);
         this.asymmetricKeyLength = Integer.valueOf(params.get(KeymanagerConstant.ASYM_KEY_SIZE));
         this.signAlgorithm  = params.get(KeymanagerConstant.CERT_SIGN_ALGORITHM);
+        this.enableKeyReferenceCache = Boolean.parseBoolean(params.get(KeymanagerConstant.FLAG_KEY_REF_CACHE));
```

Comment:
@urviljoshi What happens if this FLAG_KEY_REF_CACHE is not configured in the properties, will it automatically take it as false ?

Reply:
mosip.kernel.keymanager.keystore.keyreference.enable.cache:true this is the property it is true as default